README.md

```diff
@@ -45,9 +38,56 @@ You can check your installed versions by running the following commands from a t
 - `npm --version`
 - `docker --version`
 - `docker-compose --version`
+- `go version`
+
+## Smart Contract Development Lifecycle
+_Please note that all commands contributed by this extension are accesible via the VSCode Command Palette. The majority of the operations described are available from burger menus located on the panel headers, or by right-clicking tree items, in the extension's side bar view._
+
+The expected smart contract development lifecycle follows several broad points, all possible entirely within VSCode using this extension:
+1. Creating and packaging a smart contract
+2. Connecting to an instance of Hyperledger Fabric
+3. Running and debugging a smart contract
+
+### Create and Develop a Fabric smart contract project
+A smart contract project is a directory containing all the relevant contract and metadata files that define a smart contract. Use the `Create Smart Contract Project` command to create a basic smart contract, available in JavaScript, TypeScript, Go or Java. 
+
+### Package a smart contract project
+Once the smart contract has been created and the smart contract transactions have been written, the smart contract project must be packaged before it can be deployed. To package a project you have open in your workspace, run the `Package a Smart Contract Package` command. Packages are listed in the `Smart Contract Packages` panel. The Blockchain output channel lists what files have been packaged during this action. Alternatively run the `Import Package` command to import a pre-existing .cds package to be used within VSCode. 
+
+### Connect to the local_fabric runtime
+The extension contains a pre-configured local instance of Hyperledger Fabric named `local_fabric`, which the extension will automatically pull and use the correct Docker images for. It can be enabled and operated under the `Local Fabric Ops` panel. The first time it is started, Fabric images will be installed and an admin identity created in the `local_fabric_wallet` wallet. Left-click the `local_fabric` gateway to connect to it within the `Fabric Gateways` view, using the `admin` identity. For `local_fabric` management tasks such as restart and teardown, see the `Local Fabric Ops` panel burger menu.
```

Comment:
maybe say that the pre configured network has one, org, one peer, one channel etc

Reply:
This actually seems more to be about "Operate the local_fabric runtime", and then as Caroline suggested in another comment, it's worth a separate section that mentions using the pre-configured gateway to connect, discover instantiated smart contracts' transactions, then either submit/evaluate them in the VSCode UI or generate a test file.

Reply:
Ok, so I'm thinking of:
- Renaming this section to "Operate the local_fabric runtime"
- Renaming the "connect to another gateway" section to "Add a gateway for connecting to your own Hyperledger Fabric instance"
- And adding another section "Connect to a gateway and discover its resources" where I can explain submitting and generating tests, which is independent of what gateway you're using to do so 👍